cardano_node_tests/tests/test_scripts.py

```diff
@@ -2301,6 +2293,13 @@ def test_script_reference_utxo(
             "SimpleScript",
         )
 
+        # Check expected script type
+        if (
+            script_type_str == "SimpleScriptV1"
+            and reference_utxo.reference_script["script"]["type"] == "SimpleScriptV2"
+        ):
+            pytest.fail("Regression -> Reported 'SimpleScriptV2', see node issue #4261")
```

Comment:
**Question:**
We should remove this since is no longer applicable, or keep it and maybe a good opportunity to use https://github.com/input-output-hk/cardano-node-tests/pull/1816/ ?

Reply:
We can use the `blockers.GH...` here so we have a trail that the closed PR is/was tested.